respa/settings.py

```diff
@@ -108,6 +108,13 @@
     'django.middleware.security.SecurityMiddleware',
 ]
 
+if DEBUG:
+    INSTALLED_APPS.append('debug_toolbar')
+    INTERNAL_IPS = ['127.0.0.1', '172.20.0.1',]
```

Comment:
The 172.20.0.1 won't be the same on different machines.

Reply:
Ah.. sorry, forgot by accident. I guess we'll just have to leave it out and expect that people know to add their own IPs there.